_posts/2020-10-06-solidity-dynamic-array-cleanup-bug.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+published: true
+title:  "Solidity Dynamic Array Cleanup Bug"
+date:   2020-10-06
+author: Solidity Team
+category: Security Alerts
+---
+
+On September 17, 2020, a bug in the Solidity code generator was found. The bug is fixed with version 0.7.3
+released on October 6, 2020. The bug is present in all prior versions of Solidity.
+
+We assigned a severity level of "medium" because the bug is easy to detect while testing the contract.
```

Comment:
I would argue that this bug is one that is not trivially detectable through testing and that's why it got medium (instead of low). It is not a bug in the optimizer, so it is easier to trigger. The reasons why I would not set it to "high" are:
 - it has not been reported for several years
 - people maybe don't even rely on this behaviour and assume new array entries might be "dirty" anyway
 - increasing the length of an array without filling it with something is unlikely

Maybe we should state the severity without giving a reason for it?